docker/Dockerfile-python2.7-build

```diff
@@ -0,0 +1,85 @@
+# vim: set ft=dockerfile:
```

Comment:
I'm a little confused. How is this file different than the previous file `Dockerfile-python2.7`? Is that one for running unit tests?

Reply:
Whoops, they are exactly the same file. I didn't mean to commit the ones without `-build` or `-test`... I'll remove them.